mud_files/commissioning_tool.json

```diff
@@ -0,0 +1,220 @@
+{
+  "ietf-mud:mud": {
+    "mud-version": 1,
+    "mud-url": "https://digital-building.org/mud/commissioning_tool",
+    "last-update" : "2018-12-18T17:32:14.031Z",
+    "cache-validity": 48,
+    "is-supported": true,
+    "systeminfo": "Generic commissioning tool",
+    "from-device-policy": {
+      "access-lists": {
+        "access-list": [
+          {
+            "name" : "from-ipv4-commissioning"
+          }
+        ]
+      }
+    },
+    "to-device-policy": {
+      "access-lists": {
+        "access-list": [
+          {
+            "name" : "to-ipv4-commissioning"
+          }
+        ]
+      }
+    }
+  },
+  "ietf-access-control-list:acls": {
+    "acl": [
+      {
+        "name" : "to-ipv4-commissioning",
+        "type": "ipv4-acl-type",
+        "aces": {
+          "ace": [
+            {
+              "name": "http-todev",
+              "matches": {
+                "ietf-mud:mud": {
+                  "controller": "remote-host"
+                },
+                "ipv4": {
+                  "protocol": 6
+                },
+                "tcp": {
+                  "ietf-mud:direction-initiated": "from-device",
+                  "source-port": {
+                    "operator": "eq",
+                    "port": 80
+                  }
+                }
+              },
+              "actions": {
+                "forwarding": "accept"
+              }
+            },
+            {
+              "name": "https-todev",
+              "matches": {
+                "ietf-mud:mud": {
+                  "controller": "remote-host"
+                },
+                "ipv4": {
+                  "protocol": 6
+                },
+                "tcp": {
+                  "ietf-mud:direction-initiated": "to-device",
+                  "destination-port": {
+                    "operator": "eq",
+                    "port": 443
+                  }
+                }
+              },
+              "actions": {
+                "forwarding": "accept"
+              }
+            },
+            {
+              "name" : "sox-todev",
+              "matches" : {
```

Comment:
What is this? I think we should maybe add a "description" field that has a some info about what "sox" is.

Reply:
It is the protocol used by the Sedona framework to exchange data for commissioning between the CPT tool (commissioning interface running on Windows OS) and the EasiIO controllers, or other Sedona based controllers. More information following [this link](https://www.sedona-alliance.org/archive/doc/sox.html).